processor/probabilisticsamplerprocessor/probabilisticsampler.go

```diff
@@ -101,6 +137,11 @@ func (tsp *tracesamplerprocessor) processTraces(_ context.Context, td ptrace.Tra
 	if td.ResourceSpans().Len() == 0 {
 		return td, processorhelper.ErrSkipProcessingData
 	}
+
+	tsp.logger.Debug("Sampling policy evaluation completed",
```

Comment:
I feel like this is redundant. While this might help by comparing both metrics and logs to see if they get to the same value, this wouldn't be useful in production. How about we drop this log line?

Reply:
OK